docs/serving/convert-deployment-to-knative-service.md

```diff
@@ -0,0 +1,71 @@
+# Converting a Kubernetes Deployment to a Knative Service
+
+This topic shows how to convert a Kubernetes Deployment to a Knative Service.
+
+## Determine if your workload is a good fit for Knative
+
+In general, if your Kubernetes workload is a good fit for Knative, you can remove a lot of your manifest to create a Knative Service.
+
+There are three aspects you need to consider:
+
+- All work done is triggered by HTTP.
+- The container is stateless. All state is stored elsewhere or can be re-created.
+- Your workload uses only Secret and ConfigMap volumes.
+
+## Example conversion
+
+The following example shows a Kubernetes Nginx Deployment and Service, and shows how it converts to a Knative Service.
```

Comment:
Reference the k8s docs this was taken from?

Reply:
@evankanderson Done, thanks!